pt/lessons/specifics/mnesia.md

```diff
@@ -80,7 +89,11 @@ iex(learner@elixirschool.com)> Node.self
 :"learner@elixirschool.com"
 ```
 
-Como podemos ver agora, o nó que estamos executando é um átomo chamado `:"learner@elixirschool.com"`. Se nós executarmos `Mnesia.create_schema([node()])` novamente, nós iremos ver que ele criou uma outra pasta chamada **Mnesia.learner@elixirschool.com**. O propósito disto é bem simples. Nós em Erlang são usados para conectar a outros nós para compartilhar (distribuir) informação e recursos. Isto não tem que estar limitado a mesma máquina e pode comunicar através de LAN, internet, etc.
+Como podemos ver agora, o nó que estamos executando é um átomo chamado `:"learner@elixirschool.com"`. 
+Se nós executarmos `Mnesia.create_schema([node()])` novamente, nós iremos ver que ela criou uma outra pasta chamada **Mnesia.learner@elixirschool.com**.
+O propósito disto é bem simples. 
+Nós em Erlang são usados para conectar a outros nós para compartilhar (distribuir) informação e recursos.
+Isto não tem que estar limitado a mesma máquina e pode comunicar através de LAN, internet, etc.
 
 ## Iniciando Mnesia
 
```

Comment:
I think we are just missing to update the `Mnesia.start/0` reference in the line 100.

```diff
-```Mnesia.start/0```
+`Mnesia.start/0`
```